src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -31,6 +31,8 @@ namespace System.Collections
 #if FEATURE_NETCORE
     [FriendAccessAllowed]
 #endif
+    /// <summary>Implements the <see cref="T:System.Collections.IList" /> interface using an array whose size is dynamically increased as required.To browse the .NET Framework source code for this type, see the Reference Source.</summary>
```

Comment:
Thanks.

> For formatting the comments, I can investigate if it can be done before the code formatter too. What would be the desirable max length here?

Something around 80 or 100 is probably reasonable.

>  And do you only want to break into multiple lines if it goes past that length or do you always want to do that?

My preference would be only if it goes past that length.